src/analysis/processing/qgsalgorithmtransform.cpp

```diff
@@ -82,6 +82,11 @@ QString QgsTransformAlgorithm::shortHelpString() const
                       "Attributes are not modified by this algorithm." );
 }
 
+QString QgsTransformAlgorithm::shortDescription() const
+{
+  return QObject::tr( "Creates a vector layer with geometries reprojected to a new CRS." );
```

Comment:
```suggestion
  return QObject::tr( "Creates a vector layer with geometries transformed to a new CRS." );
```
(since it doesn't necessarily involve a map projection, eg geographic->geographic transforms)